packages/native/index.d.ts

```diff
@@ -38,6 +38,12 @@ export interface InsertRow {
   docId?: string;
   data: Uint8Array;
 }
+export const enum ValidationResult {
```

Comment:
Remove const here could helpful for removing compiler options

Reply:
This is auto generated. Not sure if there is a better option 😳

Reply:
Related: https://github.com/napi-rs/napi-rs/issues/1345

you need `--const-enum false`